README.md

```diff
@@ -1,51 +1,53 @@
 # Auth Astro
 
-Auth Astro is the Auth.js implementation for Astro by Astro´s community. 
+Auth Astro is the easiest way to add Authentication to your Astro Project. It wraps the core of [Auth.js](https://authjs.dev/) into an Astro integration which automatically adds the endpoints and handles everything else.
 
-It wraps the core Auth.js library for Astro, exposing helper methods and components to make it easy to add authentication to your app.
+(**disclaimer**: Please don´t confuse this package with [astro-auth](https://github.com/astro-community/astro-auth))
 
-## Installation
+# Installation
 
-Install the core Auth.js package as well as the auth-astro wrapper.
-
-**info:** The auth-astro wrapper will not work independently, it relies on @auth/core as a dependency.
+The easiest way to get started is adding this package using the astro cli. 
 
 ```bash
-npm install auth-astro@latest @auth/core@latest
+yarn astro add auth-astro
```

Comment:
any reason for specifing yarn? since npm is the default for most instructions I'd lean towards that

Reply:
No specific reason. I usally use yarn so it was in my head :D will change it 👍🏼